src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -262,6 +264,8 @@ export const ExpressCheckoutUI = ({ order }: ExpressCheckoutUIProps) => {
         variables: {
           input: {
             id: orderData.internalID,
+            paymentMethod: "CREDIT_CARD",
+            creditCardWalletType: "APPLE_PAY",
             buyerPhoneNumber: phone,
             buyerPhoneNumberCountryCode: null,
             shippingName: name,
```

Comment:
non-blocking: Now wondering if it is possible that an address changes from what is saved as the fulfillment type on the order somehow (like due to a bug). Will changing shipping details possibly make all the previously calculated totals wrong?

I have a similar todo in the exchange api code saying we might want to unset fulfillment type when shipping values change.